assets/src/design-system/components/contextMenu/mask.js

```diff
@@ -37,11 +37,18 @@ const ScreenMask = styled.div`
 
 export default function Mask({ onDismiss }) {
   return (
-    <ScreenMask
-      data-testid="context-menu-mask"
-      hasOnDismiss={Boolean(onDismiss)}
-      onClick={onDismiss || noop}
-    />
+    <>
+      {/*
+        TODO: Investigate
+        See https://github.com/google/web-stories-wp/issues/6671
+        */}
+      {/* eslint-disable-next-line styled-components-a11y/click-events-have-key-events, styled-components-a11y/no-static-element-interactions */}
+      <ScreenMask
+        data-testid="context-menu-mask"
+        hasOnDismiss={Boolean(onDismiss)}
+        onClick={onDismiss || noop}
```

Comment:
@samwhale do you remember why this was important? I know it is, but I can't remember precisely why.

Reply:
Yeah - the styled component chooses to block pointer events:
![image](https://user-images.githubusercontent.com/22185279/119052207-7c2a5180-b981-11eb-86b4-904306aa8b39.png)

Reply:
thank you!